libs/ui/inputs/src/lib/autocomplete/autocomplete.component.spec.ts

```diff
@@ -164,9 +184,6 @@ describe('AutocompleteComponent', () => {
       it('sends a submitted value', () => {
         expect(anyEmitted).toEqual(['bla'])
       })
-      it('closes the autocomplete panel', () => {
-        expect(component.triggerRef.closePanel).toHaveBeenCalled()
-      })
```

Comment:
This test is not needed since when we click the button the input becomes out of focus and the overlay closes anyway.